### PR TITLE
add place-name class style

### PR DESCRIPTION
**Pull request by goose-life**
Same as Act heading

Goes with https://github.com/laws-africa/indigo/pull/917

image

### CODE REVIEW

**Comment by longhotsummer** [comment]
Nice! I think this should be equivalent to h2, not h1. The place name is secondary information, the primary information is still the act name.

**Comment by goose-life** [comment]
image

**Comment by goose-life** [comment]
image

File: scss/_coverpage.scss [hunk 1]
```diff
@@ -16,6 +16,13 @@
       }
     }
 
+    .place-name {
+      font-size: ceil($akn-font-size * 1.25); // ~ 18px
+      line-height: 1.4;
```
Comment (longhotsummer) [inline scss/_coverpage.scss]: h2 has `1.2`? Do we need to change h2, or this?

Comment (longhotsummer) [reply scss/_coverpage.scss]:
```suggestion
      line-height: 1.2;
```

Comment (goose-life) [reply scss/_coverpage.scss]: Do `place-name` and `h2` need to have the same line height? 'Western Cape' and 'South Africa' are two separate things below, whereas 'Act 43 of 2011' is one thing, even if it had to run over multiple lines. At 1.2, 'Western Cape South Africa' reads as one thing.

1.2:
image

1.4:
image

Comment (longhotsummer) [reply scss/_coverpage.scss]: All the headings currently have the same line-height of 1.2. I think we should keep to that for now.

**Pull request merged**